src/libs/local_store.js

```diff
@@ -19,15 +19,31 @@ LocalStore.prototype.getAllPois = function() {
   })
 }
 
-LocalStore.prototype.register = function() {
+LocalStore.prototype.getUserInfo = function() {
+  return Promise.resolve(null)
+}
+
+LocalStore.prototype.registerApp = function() {
   console.log('local storage doesn\'t support register method')
   return Promise.resolve()
 }
 
+LocalStore.prototype.unregisterApp = function() {
+  return Promise.resolve()
```

Comment:
a think declare as async and return true / false is equivalent and more clear

Reply:
:+1: